### PR TITLE
Add pending state for proposal bridging

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Show pending state when a proposal is bridged.
- This pr is want part of adding cross chain notifications.

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #106** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/106" target="_blank">Graphite</a> 
    * **PR #110** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/110" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/110?utm_source=stack-comment).

File: components/ProposalRow.tsx [hunk 1]
```diff
@@ -34,6 +38,27 @@ export default function ProposalRow({
...
+        Waiting for relayer 🕓
+      </div>
+    ) : (
+      <button
+        className="rounded bg-indigo-600 px-2 py-1 text-xs font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+        onClick={async () => {
+          await walletClient!.switchChain({ id: l1.chain.id });
+          bridgeProposal();
+        }}
+        disabled={!bridgeProposal || bridgeProposalIsLoading}
```
Comment (garyghayrat) [inline components/ProposalRow.tsx:53]: For style consistency, we want to disable the button and make it look gray when it's clicked and pending.
image
Like this:
image

File: components/ProposalRow.tsx [hunk 2]
```diff
@@ -34,6 +38,27 @@ export default function ProposalRow({
...
+  const bridgeButton =
+    bridged !== null && !proposalMetadataBridgeIsLoading ? (
+      <div className="rounded bg-gray-500 px-2 py-1 text-xs font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
+        Waiting for relayer 🕓
+      </div>
+    ) : (
+      <button
+        className="rounded bg-indigo-600 px-2 py-1 text-xs font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+        onClick={async () => {
+          await walletClient!.switchChain({ id: l1.chain.id });
```
Comment (garyghayrat) [inline components/ProposalRow.tsx:50]: When on optimism goerli, clicking `Bridge Proposal` -> `Switch Network` on Metamask popup -> causes the following error:
image

**Review by wildmolasses — COMMENTED** [review]
this "bridge proposal" stuff may be something we can skip. In my PR #119 the button is no longer present. I'll leave the decision to you as to whether we should bring it into the new vote page, or just close.

**Comment by wildmolasses** [comment]
Closing per @alexkeating

**Pull request closed without merging**